### ISSUE TITLE
Deprecated on npm?

### ISSUE THREAD

**Issue by RobbieTheWagner** [felixrieseberg/electron-notification-state#5]
What are we supposed to use instead if this is deprecated? The Electron docs still recommend this.

**Comment by LeBlackDonut** [comment]
@rwwagner90 Did you find any alternatives? There may be node.js modules that allow us to do the same but I havent found any.

**Comment by RobbieTheWagner** [comment]
I did not find any alternatives. We are still using this.

**Comment by ferretwithaberet** [comment]
@rwwagner90 @LeBlackDonut 
There are 2 npm packages for this project, one of them is deprecated and shows the version as `1.0.4`, the other one is `1.0.0` like this project so I do not know what happened. None of them seem to be published by @felixrieseberg.
https://www.npmjs.com/package/electron-notification-state
https://www.npmjs.com/package/@meetfranz/electron-notification-state

**Comment by RobbieTheWagner** [comment]
Hmm, maybe this package is still fine to use then, it just never had an official npm release?

**Comment by hovancik** [comment]
Yeah this package is `@meetfranz/electron-notification-state` and it is not deprecated.

**Comment by bahawludba** [comment]
has anyone one found a way to use this package? I am having trouble importing this. When I build the electron app (.dmg), it is giving me a 'file name undefined' error.
